heliopy/data/mms.py

```diff
@@ -49,6 +51,10 @@ def fpi_dis_moms(probe, mode, starttime, endtime):
     fnames = []
     daylist = util._daysplitinterval(starttime, endtime)
     data = []
+    units = OrderedDict([('mms1_dis_errorflags_fast',
```

Comment:
The "mms1" bit needs to be changed depending on the probe (1 is the the probe here, but it could be 2, 3, or 4 depending on the `probe`)